include/ucdr/types/array.h

```diff
@@ -22,52 +22,35 @@ extern "C" {
 #include <ucdr/common.h>
 
 // -------------------------------------------------------------------
-//                   PUBLIC SERIALIZATION FUNCTIONS
+//                         DECLARATION MACROS
 // -------------------------------------------------------------------
+#define UCDR_ARRAY_SERIALIZE_DECLARATION(SUFFIX, TYPE) \
```

Comment:
It could be cool create an only public header and put all the public functionality there, as `gl.h` works: https://www.khronos.org/registry/OpenGL/api/GLES/1.0/gl.h

Maybe for another task

Reply:
Yeah! I like your style. We should create another task in order to handle this change.

Reply:
I created one, task 4268